src/NServiceBus.AcceptanceTests/RunDescriptorExtensions.cs

```diff
@@ -1,20 +0,0 @@
-﻿namespace NServiceBus.AcceptanceTests
-{
-    using System;
-    using AcceptanceTesting.Support;
-    using ScenarioDescriptors;
-
-    public static class RunDescriptorExtensions
-    {
-        public static Type GetTransportType(this RunDescriptor runDescriptor)
-        {
-            Type transportType;
-            if (!runDescriptor.Settings.TryGet("Transport", out transportType))
```

Comment:
yeah that's coming from the att framework and is set to `Transports.Default` if nothing else has been specified by repeat. I think there is actually a good chance of this type being set to the wrong type anyway (but doesn't actually matter).

Yep, that code will also be cleaned up on the follow up PR.